tests/model_serving/model_runtime/vllm_omni/utils.py

```
@@ -0,0 +1,286 @@
+        voice = _voice_cache.get(url) or discover_tts_voice(isvc=isvc)
+        _voice_cache[url] = voice
+    model_name = isvc.instance.metadata.name
+    response = requests.post(
+        f"{url}/v1/audio/speech",
+        json={
+            "model": model_name,
+            "input": text,
+            "voice": voice,
+            "response_format": response_format,
+        },
+        verify=False,
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)